### PR TITLE
Additional info for auditors

### PR DESCRIPTION
**Pull request by mogul**

### CODE REVIEW

File: content/docs/compliance/customer-separation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Garden uses the [Open Container 
+Initiative](https://www.opencontainers.org/)'s standard [runC](https://github.com/opencontainers/runc) implementation,
+and applies [safeguards](https://docs.cloudfoundry.org/concepts/architecture/garden.html#garden-runc) above and beyond normal Docker or Kubernetes environments that fit the multi-tenant PaaS context. 
+Cloud Foundry provides [additional details about various aspects of container security](https://docs.cloudfoundry.org/concepts/container-security.html). 
+[Docker images are further restricted](https://docs.cloudfoundry.org/adminguide/docker.html#multi-tenant) when running in Cloud Foundry to protect customers from each others. As a result, containers on cloud.gov are highly secure by default. 
+
+The only "moving piece" the cloud.gov team controls in our deployment is [whether privileged containers are used for applications or for buildpack staging processes](https://docs.cloudfoundry.org/concepts/container-security.html#types). 
+Both of these options are set to `false` in cloud.gov. Please contact us if you need to verify this setting in our system, or have further questions about how customer separation works in cloud.gov.
```
Comment (mogul) [inline content/docs/compliance/customer-separation.md]: @18F/cloud-gov-ops is there a way for assessors to inspect this setting themselves in version controlled manifests, or via a runtime test?

File: content/docs/compliance/customer-separation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+menu:
+  docs:
+    parent: compliance
+title: Customer separation
```
Comment (brittag) [inline content/docs/compliance/customer-separation.md:5]: Suggest we also add cross-references to where we discuss these things in the SSP, to help people find more details (and so they can verify this page against our 3PAO-audited/JAB-reviewed info, if they need to). Probably good to cross-reference some or all of section 9.7, section 10, CM-7 (2), SC-39, SI-16.

Comment (mogul) [reply content/docs/compliance/customer-separation.md:5]: Is there a precedent for how we do that, content-wise?

Comment (brittag) [reply content/docs/compliance/customer-separation.md:5]: I don't think we have a really good precedent yet - we have a basic note at the end of the section at https://cloud.gov/docs/getting-started/app-maintenance/#operating-system-patching - something like that would be a good start, and we can make it better later as needed.

**Comment by mogul** [comment]
@brittag look good? If I can merge this early Friday I'll refer to it live rather than in the pull-request in further comms.

**Review by brittag — APPROVED** [review]
@mogul I did some copyediting for clarity, and I updated the list of controls - please take a look and merge if it looks good to you!

**Pull request merged**